eventDataScript.js

```diff
@@ -0,0 +1,296 @@
+    if (input === '') return;
+
+    var season = getCookie('currCheckedSeason');
+    var currEvent = getCurrEvent(); 
+    var cname = event_data_cookie_name(season,currEvent);
+    var teamsJSON = JSON.parse(getCookie(cname));
+
+    delete teamsJSON[input];
+    setCookie(cname,JSON.stringify(teamsJSON),750);
+    setTeamTable(season,currEvent);
+    $("#deleteSpecificTeamTextBox").val("").focus();
+}
+
+$(document).ready(function(){
+    loadPage();
+
+    const Enter_key_code = 13;
+    $('#newEventInputBox').keypress(function(e){
+        if (e.keyCode == Enter_key_code) addNewEvent();
+    });
+
+    //event specific teams code
+    $('#teamName').keypress(function(e){
+        if (e.keyCode == Enter_key_code) $('#teamSaveButton').click();
+    });
+
+    $('#deleteSpecificTeamTextBox').keypress(function(e){
+        if (e.keyCode == Enter_key_code) deleteSpecificTeam();
+    });
+
+    $('#teamSaveButton').click(function(){
+        var currSeason = getCookie('currCheckedSeason');
+        var currEvent = getCurrEvent();
+        var teamNumber = parseInt($("#teamNumber").val().trim());
+        var teamName = $("#teamName").val().trim();
+        var teams = {};
+
+        if (teamNumber === "" || teamName === "")
+        {
+            $('#teamConfirmationBox').html("<i>Please fill in both fields.<\i>");
+            return;
+        }
+
+        if (currSeason === ' ')
+        {
+            $("#teamConfirmationBox").html("<i>Please check a season.<\i>");
+            return;
+        }
+
+        if (currEvent === ' ')
+        {
+            $("#teamConfirmationBox").html("<i>Please check an event.<\i>");
+            return;
+        }
+
+        var cname = event_data_cookie_name(currSeason,currEvent);
+        var teams;
+        var raw_stored_data = getCookie(cname).trim();
+        if (!raw_stored_data == '') //if cookie is stored yet
+        {
+            teams = JSON.parse(raw_stored_data);
+        } else //cookie not stored yet
+        {
+            teams = {};
+        }
+        teams[teamNumber] = teamName;
+        setCookie(cname,JSON.stringify(teams),750);
+        setTeamTable(currSeason,currEvent);
+        
+        $("#teamConfirmationBox").html('<i>Team ' + teamNumber + ': ' + teamName + ' has been added.<\i>');
+        $("#teamNumber").val("").focus();
+        $("#teamName").val("");
+    });
+});
+
+// helper methods below
+
+function deleteCookie(name)
+{
+    document.cookie = name + '=;expires=Thu, 01 Jan 1970 00:00:00 UTC;path=/';   
+}
+
+function setCookie(cname, cvalue, exdays=750) {
+    var d = new Date();
+    d.setTime(d.getTime() + (exdays*24*60*60*1000));
+    var expires = "expires="+ d.toUTCString();
+    document.cookie = cname + "=" + cvalue + ";" + expires + ";path=/";
+}
+
+function getCookie(name)
+{
+    var re = new RegExp(name + "=([^;]+)");
+    var value = re.exec(document.cookie);
+    return (value != null) ? unescape(value[1]) : " ";
+}
```

Comment:
put these three methods in a common js file (maybe call it cookieHelpers.js) and include it from the html rather than copy and pasting it to multiple pages.